packages/api/src/middleware/validation/validateWebhook.js

```diff
@@ -0,0 +1,17 @@
+async function validateRequest(req, res, next) {
+  const partnerId = req.params.partner_id;
+  if (partnerId !== 1) {
+    next();
+  } else {
+    const farmId = req.params.farm_id;
```

Comment:
I think you removed the farm_id from the params, or maybe this happened when you merged collab into this branch